dashboard/app/models/pd/payment/payment_calculator_district.rb

```diff
@@ -0,0 +1,19 @@
+module Pd::Payment
+  # Payment model for district workshops.
+  class PaymentCalculatorDistrict < PaymentCalculatorBase
+    PAYMENT_PER_TEACHER_PER_DAY = 40
```

Comment:
Is this the payment for teacher or a food allowance? It's hard to infer from context and comments